src/subcommand/flow.rs

```diff
@@ -0,0 +1,113 @@
+use {super::*, base64::Engine, bitcoin::psbt::Psbt};
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct Output {
+  inputs: Vec<Vec<Range>>,
+  outputs: Vec<Vec<Range>>,
+  fee: Vec<Range>,
+}
+
+#[derive(Debug, Parser)]
+pub(crate) struct Flow {
+  #[arg(long)]
+  binary: bool,
+  psbt: PathBuf,
+}
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct Range {
+  start: u64,
+  end: u64,
+  name: String,
+}
+
+impl From<(u64, u64)> for Range {
+  fn from((start, end): (u64, u64)) -> Self {
+    Self {
+      start,
+      end,
+      name: Sat(start).name(),
+    }
+  }
+}
+
+impl Flow {
+  pub(crate) fn run(self, settings: Settings) -> SubcommandResult {
+    let psbt = if self.binary {
+      fs::read(self.psbt).unwrap()
+    } else {
+      base64::engine::general_purpose::STANDARD
+        .decode(fs::read_to_string(self.psbt).unwrap().trim())
+        .unwrap()
+    };
```

Comment:
when trying to run this I was doing `ord flow <base64 psbt>` maybe if this file doesn't exist it should try to parse `self.psbt` to base 64